config/config.go

```diff
@@ -18,10 +20,12 @@ var httpClient = &http.Client{
 }
 
 type Config struct {
-	CMD          CMD           `yaml:"cmd"`
-	Environments []Environment `yaml:"environments"`
-	SSHUser      string        `yaml:"ssh-user"`
-	DPSetupPath  string        `yaml:"dp-setup-path"`
+	CMD                    CMD           `yaml:"cmd"`
+	Environments           []Environment `yaml:"environments"`
+	SSHUser                string        `yaml:"ssh-user"`
+	DPSetupPath            string        `yaml:"dp-setup-path"`
+	DPHierarchyBuilderPath string        `yaml:"dp-hierarchy-builder-path"`
+	DPCodeListScriptsPath  string        `yaml:"dp-code-list-scripts-path"`
```

Comment:
Is it worth putting these two paths under the CMD config section, as they are specific to CMD?

Reply:
I debated the same, not really sure which is best but leaned towards the consistency. Happy to change if there are strong views the other way?